governance/program/src/error.rs

```diff
@@ -162,20 +162,6 @@ pub enum GovernanceError {
     #[error("Invalid Signatory Mint")]
     InvalidSignatoryMint,
 
-    /// ---- Account Tools Errors ----
-
-    /// Invalid account owner
-    #[error("Invalid account owner")]
-    InvalidAccountOwner,
-
-    /// Account doesn't exist
-    #[error("Account doesn't exist")]
-    AccountDoesNotExist,
-
-    /// Invalid Account type
-    #[error("Invalid Account type")]
-    InvalidAccountType,
-
```

Comment:
Will removing these cause any issues for the programs already deployed?

Reply:
If there were any programs using the error codes then yes, but I'm not aware of any.
For the UI side I'll make a change to accommodate the breaking change.